fast_api_als/routers/submit_lead.py

```diff
@@ -139,6 +162,8 @@ async def submit(file: Request, apikey: APIKey = Depends(get_api_key)):
     # score the lead
     result = score_ml_input(ml_input, make, dealer_available)
 
+    t3 = [int(time.time() * 1000.0)]
```

Comment:
you could have updated t1, t2 instead of creating t3.
because we aren't using these time variables anywhere in our code later.